src/rulesets/operations.ts

```diff
@@ -31,22 +31,24 @@ const preventParameterChange = (schemaProperty: string) => {
 
 export const rules = {
   operationId: ({ operations }: SnykApiCheckDsl) => {
-    operations.requirementOnChange.must(
-      "have the correct operationId format",
-      (operation, context, docs) => {
-        docs.includeDocsLink(links.standards.operationIds);
-        docs.becomesEffectiveOn(new Date("2021-07-01"));
-        expect(operation.operationId).to.be.ok;
-        if (operation.operationId !== undefined) {
-          const normalized = camelCase(operation.operationId);
-          expect(
-            normalized === operation.operationId &&
-              prefixRegex.test(operation.operationId),
-            `operationId "${operation.operationId}" must be camelCase (${normalized}) and start with get|create|list|update|delete`,
-          ).to.be.ok;
-        }
-      },
-    );
+    operations.requirementOnChange
+      .attributes("operationId")
+      .must(
```

Comment:
constrain the operation id naming rule to changes to its operationId

**Currently** if you changed changed anything else in the operation it would run this rule 

**Now**  this only runs if the operationId actually changed